src/core/decoder/src/decode/huffman.c

```diff
@@ -21,15 +21,15 @@
 
 /*- General utility functions -------------------------------------------------------------------*/
 
-static inline uint8_t clz(int32_t streamData, uint8_t numBits)
+static inline uint8_t clz(uint32_t streamData, uint8_t numBits)
```

Comment:
Why this change? We're using `clz` on `uint8_t` data generally, and all three `clz` functions expect `int32_t`s I thought?

Reply:
Ah wait no, you're right, they all expect `uint32_t`s, good call.